src/api/client/Identities.ts

```diff
@@ -65,6 +74,20 @@ export class Identities {
    */
   public registerIdentity: ProcedureMethod<RegisterIdentityParams, Identity>;
 
+  /**
+   * Register an Identity and create a CDD claim for it. This allows for an Account to receive POLYX and interact with the chain.
+   *
+   * Functions like {@link api/client/Identities!registerIdentity | registerIdentity} followed by a CDD claim being added, except in a single transaction.
+   *
+   * @note this may create {@link api/entities/AuthorizationRequest!AuthorizationRequest | Authorization Requests} which have to be accepted by the `targetAccount`.
+   *   An {@link api/entities/Account!Account} or {@link api/entities/Identity!Identity} can fetch its pending Authorization Requests by calling {@link api/entities/common/namespaces/Authorizations!Authorizations.getReceived | authorizations.getReceived}.
+   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link api/entities/common/namespaces/Authorizations!Authorizations.getOne | authorizations.getOne}
+   *
+   * @note required role:
+   *   - Customer Due Diligence Provider
+   */
+  public registerIdentityWithCdd: ProcedureMethod<RegisterIdentityWithCddParams, Identity>;
```

Comment:
Any reason for adding a new method instead of overloading the existing one? No preference, but I'd like to know your thoughts